frontend/package.json

```diff
@@ -27,7 +27,10 @@
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
-    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-devconsole": "cd packages/dev-console/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-devconsole-headless": "cd packages/dev-console/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser chrome --headless --spec \"features/project-creation.feature\";",
+    "test-cypress-console-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-headless": "cd ../../ && yarn run test-cypress-console-headless && yarn run test-cypress-devconsole-headless",
```

Comment:
Does this work?  The starting dir should be `frontend`, so `../../` would be parent dir of `console`!?
I was thinking it would be:

"test-cypress-devconsole-headless": "cd packages/dev-console/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser chrome --headless --spec \"features/project-creation.feature\" **&& cd ../../../**",  <-- return to `frontend` from `packages/dev-console/integration-tests`

"test-cypress-console-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless **&& cd ../../**", <-- return to `frontend` from `packages/integration-tests-cypress`

"test-cypress-headless": "yarn run test-cypress-console-headless && yarn run test-cypress-devconsole-headless",

Reply:
updated

Reply:
Hi, this doesn't seem to work: 
```
frontend>yarn run test-cypress-headless
yarn run v1.21.0
$ cd ../../ && yarn run test-cypress-console-headless && cd ../../../ && yarn run test-cypress-devconsole-headless
error Couldn't find a package.json file in "/home/dtaylor/repos/openshift/origin/src/github.com/openshift"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```
It looks like cd'ing via `cd foobar && yarn run ...` is temparary, just for that line, so we don't need to cd back to `frontend`.   

This seems to work:
`"test-cypress-headless": "yarn run test-cypress-console-headless && yarn run test-cypress-devconsole-headless"`
...sorry for the confussion on this :-)

Reply:
updated, please review